k8s_test.go

```diff
@@ -136,3 +144,44 @@ func (s *SuiteKubernetesCluster) Test_ReplaceKubernetesNode() {
 	s.NoError(err)
 	s.NotNil(rsp)
 }
+
+func (s *SuiteKubernetesCluster) Test_AutoscalingEnabled() {
+	s.Run("enabled true", func() {
+		autoscaling := AutoScaling{
+			MinNodeCount: &zero,
+			MaxNodeCount: &five,
+		}
+		s.True(autoscaling.Enabled())
+		autoscaling = AutoScaling{
+			MinNodeCount: &one,
+			MaxNodeCount: &five,
+		}
+		s.True(autoscaling.Enabled())
+	})
+	s.Run("enabled false, autoscaling undefined", func() {
+		var autosscaling AutoScaling
+		s.False(autosscaling.Enabled())
+
+	})
+	s.Run("enabled false, limits 0", func() {
+		autoscaling := AutoScaling{
+			MinNodeCount: &zero,
+			MaxNodeCount: &zero,
+		}
+		s.False(autoscaling.Enabled())
+	})
+	s.Run("enable false, minnodecount nil", func() {
```

Comment:
be consistent with the subtest names: enable vs enabled, same for the test below